ipyopenlayers/openlayers.py

```diff
@@ -259,96 +135,91 @@ class Map(DOMWidget):
     _view_module_version = Unicode(module_version).tag(sync=True)
 
     center = List(def_loc).tag(sync=True, o=True)
-    zoom = CFloat(0).tag(sync=True, o=True)
+    zoom = CFloat(2).tag(sync=True, o=True)
     layers = List(Instance(Layer)).tag(sync=True, **widget_serialization)
     overlays=List(Instance(BaseOverlay)).tag(sync=True, **widget_serialization)
     controls=List(Instance(BaseControl)).tag(sync=True, **widget_serialization)
 
 
 
     def __init__(self, center=None, zoom=None, **kwargs):
-        """Initialize the Map with optional center and zoom level.
-
-        Parameters
-        ----------
-        center: list of float, optional
-            The initial center of the map.
-        zoom: float, optional
-            The initial zoom level of the map.
-        """
         super().__init__(**kwargs)
+        self._click_callbacks = []
         if center is not None:
             self.center = center
         if zoom is not None:
             self.zoom = zoom
 
-    def __repr__(self):
-        """Return a string representation of the Map instance."""
-        return f"Map(center={self.center}, zoom={self.zoom})"
-    
     def add_layer(self, layer):
-        """Add a layer on the map.
-
-        Parameters
-        ----------
-        layer: Layer instance
-            The new layer to add.
-        """
         self.layers = self.layers + [layer]
 
     def add_overlay(self, overlay):
-        """Add an overlay to the map.
-
-        Parameters
-        ----------
-        overlay: BaseOverlay instance
-            The overlay to add.
-        """
         self.overlays = self.overlays + [overlay]
 
     def remove_layer(self, layer):
-        """Remove a layer from the map.
-
-        Parameters
-        ----------
-        layer: Layer instance
-            The layer to remove.
-        """
         self.layers = [x for x in self.layers if x != layer]
     
     def remove_overlay(self, overlay):
-        """Remove an overlay from the map.
```

Comment:
Your PR is removing the docstrings, maybe due to a wrong rebase

Reply:
yes, fixed thanks